gitpod-io/website
# ✨ New `/solutions/platform-teams` page

## Description



This PR introduces the new page specifically about Gitpod for Platform Teams.

One thing, we need to decide on is that under route (URL), this page should be there?

|Options|~🅰 (`/for/platform-teams`)~|🅱 ( `/solutions/platform-teams`) ✅ |
|:---:|:---:|:---:|
|Description| ~* We need to figure out the callout (footer/Navbar/etc) for this page <br/> * This is not ideal to have a `/for` page for a particular team~| * This is more ideal place to be in. BUT, we need to change the `platform-teams` in URL and could have something like `secure-platforms` or so...|

## Related Issue(s)



Fixes MKWEB-29

## How to test



- Open [Preview](https://deploy-preview-3737--gitpod-kumquat.netlify.app/)
- for now, Go to [`/solutions/platform-teams`](https://deploy-preview-3737--gitpod-kumquat.netlify.app/solutions/platform-teams) & see :eyes:

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing



## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.


  </details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3737"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
